src/DocumentationViewer.js

```diff
@@ -0,0 +1,219 @@
+import GObject from "gi://GObject"
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio"
+import GLib from "gi://GLib";
+import Adw from "gi://Adw";
+import WebKit from "gi://WebKit"
+
+import resource from "./DocumentationViewer.blp";
+
+Gio._promisify(
+  WebKit.WebView.prototype,
+  "evaluate_javascript",
+  "evaluate_javascript_finish"
+);
+
+Gio._promisify(
+  Gio.File.prototype,
+  "enumerate_children_async",
+  "enumerate_children_finish",
+);
+
+Gio._promisify(
+  Gio.FileEnumerator.prototype,
+  "next_files_async",
+  "next_files_finish",
+);
+
+Gio._promisify(
+  Gio.File.prototype,
+  "load_contents_async",
+  "load_contents_finish",
+);
+
+export default function DocumentationViewer({
+  window:application_window,
+  application
+  }) {
+  const builder = Gtk.Builder.new_from_resource(resource);
+  const window = builder.get_object("documentation_viewer");
+
+  const container = builder.get_object("webview_container");
+  const webview = new WebKit.WebView();
+  const listbox = builder.get_object("listbox");
+
+  const sidebar_button = builder.get_object("sidebar_button");
+  const search_button = builder.get_object("search_button");
+  const search_entry = builder.get_object("search_entry");
+  const search_bar = builder.get_object("search_bar");
+  const back = builder.get_object("back");
+  const forward = builder.get_object("forward");
```

Comment:
```suggestion
  const back = builder.get_object("button_back");
  const forward = builder.get_object("button_forward");
```

Reply:
Back button should be disabled when webview.can_go_back is false

Forward button should be disabled when webview.can_go_forward is false

Please use property bindings for that

Reply:
I've made the change but I haven't used property bindings because ``can_go_back`` / ``can_go_forward`` are functions
https://github.com/sonnyp/Workbench/pull/358/commits/23b9f60498d0107d0c48a1aba3c14281302928c0#diff-6533bf97682b99bf434791a454793569a69668f437288ddcedac958454abf4a7R36-R37